maths/maths.py

```diff
@@ -0,0 +1,112 @@
+
+# package maths
+# Implement some 
+# trigonometric algorithm
+
+Pi = 3.14159265358979323846
+
+
+def _abs(value):
+    # >>> _abs(-10)
+    # 10
+    # >>> _abs(10)
+    # 10
```

Comment:
```suggestion
    """
    >>> _abs(-10)
    10
    >>> _abs(-.5)
    0.5
    >>> _abs(0)
    0
    >>> _abs(.5)
    0.5
    >>> _abs(10)
    10
    """
```
Doctests must be in docstrings, not in comments.  Please do `python3 -m docstring -v maths.py` on your computer and make sure that all tests run and pass.